base/java-tools/src/com/netscape/cmstools/cli/MainCLI.java

```diff
@@ -465,6 +466,8 @@ public void parseOptions(CommandLine cmd) throws Exception {
         String messageFormat = cmd.getOptionValue("message-format");
         config.setMessageFormat(messageFormat);
         logger.info("Message format: " + messageFormat);
+
+        optionsParsed = true;
```

Comment:
`optionsParsed` is a new variable introduced in this patch. This line sets it to `true`, but no logic to set it to `false`?

Reply:
Boolean attributes are `false` by default, so it should be OK.

Reply:
Endi tells me to quit writing `boolean optionsParsed = false` since that's the default... :)

Reply:
The initialization is redundant for primitive class attributes:
* https://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5
* http://www.javapractices.com/topic/TopicAction.do?Id=14

Reply:
huh?! I never knew. TIL!